### PR TITLE
Add Translation API

### PR DESCRIPTION
**Pull request by Angel-AG**
Translate comments into any supported language.

### CODE REVIEW

**Review by mschillaci — APPROVED** [review]
Looks good to me!

File: portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java [hunk 1]
```diff
@@ -110,6 +121,25 @@ private Query buildQuery(String commentPicId, boolean isDateDescending) {
...
+  private void translateComments(List<Comment> comments, String targetLang) {
+    try {
+      Translate translate = TranslateOptions.getDefaultInstance().getService();
+      
+      for (Comment comment : comments) {
+        Translation translation = translate.translate(
+            comment.getComment(),
+            Translate.TranslateOption.targetLanguage(targetLang));
+
+        comment.setComment(translation.getTranslatedText());
```
Comment (akihiroota87) [inline portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java]: Nit: comment.setComment() is a little repetitive. What do you think about comment.setText()?

**Review by akihiroota87 — APPROVED** [review]
Awesome work! Just left one tiny comment, but don't want to block your progress :)

File: portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java (hunk 1 above)
Comment (Angel-AG) [reply portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java]: Sure, sounds good!

**Pull request merged**